### ISSUE TITLE
Form fields misaligned on create page

### ISSUE THREAD

**Issue by kmac420** [0xfe/pennywall#5]
On https://pennywall.io/create/index.html
Using Google Chrome - Version 72.0.3626.109 
<img width="1430" alt="pennywall-create-form-fields" src="https://user-images.githubusercontent.com/474476/53132636-e9967000-3535-11e9-8efe-cd6f56189ead.png">

**Comment by 0xfe** [comment]
I can't reproduce this. Does this happen in incognito mode too?

**Comment by kmac420** [comment]
Yes, in chrome and incognito mode. Make sure your browser window is full width as the problem goes away when the window is made narrower. I can't reproduce the error in Safari.

**Comment by 0xfe** [comment]
Darnit, I can't reproduce. :-/
